source/contracts/Controlled.sol

```diff
@@ -8,6 +8,9 @@ import 'libraries/token/ERC20Basic.sol';
 
 contract Controlled is IControlled {
     IController internal controller;
+    // Inidcates whether ETH may be extracted from this contract by the controller. It should only be allowed when ETH cannot legitimately be held by the contract
+    bool internal mayExtractETH = true;
```

Comment:
Since `mayExtractETH` is the "less secure" option, we should probably default to `false` and then have contracts explicitly assert that ETH extraction is reasonable.  This way if we "forget" one, we don't introduce an attack vector for the controller owner.